hub/powertoys/registry-preview.md

```diff
@@ -39,3 +39,5 @@ After opening a Windows Registry file, the file content is shown on the left sid
 **Reload from file**: Select `Reload from file` to reload file content if the loaded file is being edited outside of the **Registry Preview**.
 
 **Write to Registry**: Select `Write to Registry` to save any changes listed in the loaded file to the Windows Registry. The Windows Registry Editor will open and ask for confirmation before writing changes. The User Account Control (UAC) prompt, a Windows security feature designed to mitigate the impact of malware, will appear and need approval to open Windows Registry Editor.
+
+**Open key...**: Select `Open key` to open the Windows Registry Editor with whatever key you have highlighted in the right side of the application as the initial starting point.
```

Comment:
@randyrants Interaction with UI elements should use **bold** text, not `code`. And this might look better in a table.

I think this will be nice to have too: https://marketplace.visualstudio.com/items?itemName=docsmsft.docs-authoring-pack and have a look at https://learn.microsoft.com/contribute/text-formatting-guidelines :)

Reply:
The guidelines will be helpful: my initial draft of this doc was re-written by someone with a much better understanding of what the layout is.  With that in place, I've just been following their guidance and adding a new feature that I added to the program side of the house.  Having the overall text formatting guidelines will help for certain.

The plugin is nifty too, but I do most of my MD markup by hand, since I use the full VS IDE, rather than Code.  Still, every tool can help.  Might be time to install it!